ci/terraform/oidc/api-gateway.tf

```diff
@@ -576,3 +576,47 @@ EOF
     aws_api_gateway_method.robots_txt_method,
   ]
 }
+
+resource "aws_api_gateway_resource" "orch_frontend_resource" {
+  rest_api_id = aws_api_gateway_rest_api.di_authentication_api.id
+  parent_id   = aws_api_gateway_rest_api.di_authentication_api.root_resource_id
+  path_part   = "orch-frontend"
```

Comment:
We're probably going to need to think about our pathnames and routing on the frontend, unless this strips out this path segment?

Reply:
The path segment does get stripped